docker/Makefile

```diff
@@ -14,7 +14,7 @@
 
 REGISTRY?=docker.io
 IMAGE_NAME=controller
-IMAGE_VERSION?=v0.0.1
+IMAGE_VERSION?=v0.0.0
```

Comment:
setting it to v0.0.0 so that when we cut release, we can update it to v0.0.1 to trigger staging image push detecting changes in the `docker/Makefile`

Reply:
This change will trigger build of `v0.0.0` staging image which we don't want.

Reply:
I'm going back and forth on this. If we don't change this, once this PR is merged it'll trigger the image build. And I'm hoping to do v0.0.1 with https://github.com/kubernetes-sigs/secrets-store-sync-controller/pull/58.